File: app/src/main/java/ru/vegax/xavier/a3test/models/UserPhotoWrapper.java
package ru.vegax.xavier.a3test.models;

import java.util.List;

public class UserPhotoWrapper {


    private List<UserPhoto> mList;
    private Throwable mThrowable;

    public UserPhotoWrapper(List<UserPhoto> list) {
        mList = list;
    }

    public UserPhotoWrapper(Throwable throwable) {
        mThrowable = throwable;
    }

    public List<UserPhoto> getList() {
        return mList;
    }

    public Throwable getThrowable() {
        return mThrowable;
    }

}

File: app/src/main/java/ru/vegax/xavier/a3test/models/UserPhotoWrapper.kt
package ru.vegax.xavier.a3test.models

class UserPhotoWrapper {


    var list: List<UserPhoto>? = null
    var throwable: Throwable? = null

    constructor(list: List<UserPhoto>?) {
        this.list = list
    }

    constructor(throwable: Throwable) {
        this.throwable = throwable
    }
}


